quickstarts/ansible-automation-controller/config.yml

```
@@ -0,0 +1,45 @@
+
+
+slug: ansible-automation-controller
+description: |
+  ## Why monitor Ansible Automation Controller?
+  An ideal Ansible Automation Controller performance monitor must offer comprehensive and actionable information that software developers need to troubleshoot an application successfully. Some key components are:
```

I think this be less passive. For example:

```suggestion
  Use New Relic to monitor your Ansible Automation Controller's performance, so that you can get comprehensive and actionable information that your developers can use to successfully troubleshoot problems. Some key capabilities are:
```

---

Hi @**paperclypse** ,
Reviewed text correction has been changed. Kindly review it.